config/routes.rb

```diff
@@ -14,29 +14,37 @@
     resources :salespeople_invitations, only: [:index, :create, :destroy] do
       get 'resend_invitation', on: :member
     end
+
     resources :agent_invitations, only: [:new, :create] do
       get 'search_agents', on: :collection
     end
+
     resources :wholesalers, only: [:edit, :update]
     resources :agencies, only: [:edit, :update]
     resources :client_invitations, only: [:new, :create]
+
     resources :agent_wholesaler_applications, only: [:index, :destroy] do
       get 'agent_templates', on: :member
     end
-    resources :insurance_applications, only: [:index, :edit, :update, :show], path: 'applications'
+
+    resources :insurance_applications, only: [:index, :edit, :update, :show], path: 'insurance-applications'
```

Comment:
why change path?  my understanding is that we're using 'insurance-applications' internally because we're build an 'application' so we need to make a distinction.  However, the user's frame of reference is just 'application' and doesn't need clarity.